perun-oidc-server/src/main/java/cz/muni/ics/oidc/server/claims/sources/Ga4ghApiClaimSource.java

```diff
@@ -0,0 +1,81 @@
+package cz.muni.ics.oidc.server.claims.sources;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import cz.muni.ics.oidc.server.claims.ClaimSource;
+import cz.muni.ics.oidc.server.claims.ClaimSourceInitContext;
+import cz.muni.ics.oidc.server.claims.ClaimSourceProduceContext;
+import cz.muni.ics.oidc.server.claims.ClaimUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Collections;
+import java.util.Set;
+
+@Slf4j
```

Comment:
Please add configuration into the class JavaDoc. Also, it would be possible to move the ClaimSource to the GA4GH package.